plugins/mygene/src/index.ts

```diff
@@ -0,0 +1,357 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import AdapterType from '@jbrowse/core/pluggableElementTypes/AdapterType'
+import PluginManager from '@jbrowse/core/PluginManager'
+import Plugin from '@jbrowse/core/Plugin'
+import {
+  BaseFeatureDataAdapter,
+  BaseOptions,
+} from '@jbrowse/core/data_adapters/BaseAdapter'
+import { Region } from '@jbrowse/core/util/types'
+import { ObservableCreate } from '@jbrowse/core/util/rxjs'
+import SimpleFeature, { Feature } from '@jbrowse/core/util/simpleFeature'
+import AbortablePromiseCache from 'abortable-promise-cache'
+import QuickLRU from '@jbrowse/core/util/QuickLRU'
+import {
+  readConfObject,
+  ConfigurationSchema,
+} from '@jbrowse/core/configuration'
+import { AnyConfigurationModel } from '@jbrowse/core/configuration/configurationSchema'
+
+const configSchema = ConfigurationSchema(
+  'MyGeneV3Adapter',
+  {
+    baseUrl: {
+      type: 'string',
+      defaultValue:
+        // eslint-disable-next-line no-template-curly-in-string
+        'https://mygene.info/v3/query?q=${ref}:${start}-${end}&size=1000&fields=all&size=1000&email=colin.diesh@gmail.com&species=human',
```

Comment:
Do you want your personal email here?

Reply:
hmm...it's a good question, maybe I will put it in the config.json...I just have had them in the past alert me if my plugin is acting up, but on some level it's hard to respond to that if it's deployed on not-my-site

Reply:
removed email for now